internal/engine/buildcontrol/target_queue.go

```diff
@@ -90,8 +90,24 @@ func NewImageTargetQueue(ctx context.Context, iTargets []model.ImageTarget, stat
 	}, nil
 }
 
-func (q *TargetQueue) Results() store.BuildResultSet {
-	return q.results
+// New results that we built with a BuildHandler. Omits results
+// that were re-used previous builds.
```

Comment:
I don't care HOW we built them (aren't all results built with a BuildHander?), I care WHEN we built them. "Results that were build in the current call (omits results...)" or something?

Reply:
done